ddtrace/mocktracer/mocktracer.go

```diff
@@ -27,6 +27,9 @@ var _ Tracer = (*mocktracer)(nil)
 
 // Tracer exposes an interface for querying the currently running mock tracer.
 type Tracer interface {
+	// OpenSpans returns the set of started spans that have not been finished yet.
+	OpenSpans() []Span
```

Comment:
Cool. I think this is a good addition. Could be a good fail-safe to test on most integrations that no unfinished spans are left!